solutuins-1/src/test/java/solutions_1/solutuins_1/NonReducibleTest.java

```diff
@@ -0,0 +1,59 @@
+package solutions_1.solutuins_1;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Test;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * The Class PrimeStringTest.
+ */
+
+
+@RunWith(Parameterized.class)
+public class NonReducibleTest {

```

Comment:
Similar to KattapaReadyForBattle test class, please update the tests other classes.

Reply:
Sure, will do and commit the changes